cmd/convox/uninstall.go

```diff
@@ -91,7 +91,7 @@ func cmdUninstall(c *cli.Context) error {
 
 	// verify that rack was detected
 	if len(stacks.Rack) == 0 || stacks.Rack[0].StackName != rackName {
-		return stdcli.Error(fmt.Errorf("can not find rack named %s", rackName))
+		return stdcli.Error(fmt.Errorf("can not find stack named %s\nAre you authenticating with the correct AWS account?\nSee AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and ~/.aws/credentials", rackName))
```

Comment:
Happy to revert the change. I switched to "stack" to match the CLI help text for `uninstall`:

```
❯ convox help uninstall
convox uninstall: uninstall a convox rack

Usage:
  convox uninstall <stack-name> <region> [credentials.csv]
...
```

Seems like we kind of force users to be familiar with the implementation details here already. I will switch it back and let others discuss it later, though.